bin/run-tests

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -x
+
+base="bundle exec rspec -f progress -f JUnit -o spec/ruby-agent-junit.xml"
+specific_spec=$1
+
+if [[ $specific_spec = '' ]]; then
+  echo 'Running all specs, including integration'
+
+  $base
+  find spec/integration -name '*_spec.rb' -exec $base {} \;
```

Comment:
it launches several tests in a sequential way, each one will generate a junit file, if all use the same name we will only have the latest one, so we have to make the name uniq for each test, we would need a function